docs/advanced-topics/verification.rst

```diff
@@ -0,0 +1,126 @@
+    etc
+8.  Select the “verified” status either as “true” or “false”. Use “true”
+    to verify, “false” to “unverify”.
+9.  Provide “metadata”, this is typically the verifier’s identity which
+    can be something like “GA4GH/DREAM Challenge”
+10. Lastly, provide your Dockstore token using the lock icon at the top
+    right of the endpoint
+
+Below is a screenshot of someone verifying the “test.json” test
+parameter file of the “master” version of the “dockstore-tool-md5sum”
+tool.
+
+.. figure:: /assets/images/docs/verification/swagger.png
+   :alt: Swagger-UI
+
+   Swagger-UI
+
+The curl command results in something like:
+
+::
+
+   curl -X POST "https://dockstore.org/api/api/ga4gh/v2/extended/quay.io%2Fbriandoconnor%2Fdockstore-tool-md5sum/versions/master/CWL/tests/test.json?platform=Dockstore%20CLI&verified=true&metadata=Phase%201%20GA4GH%20Tool%20Execution%20Challenge" -H  "accept: application/json" -H  "Authorization: Bearer iamafakebearertoken"
+
+A successful response will result in something like:
+
+::
+
+   {
+     "Dockstore CLI": {
+       "metadata": "Phase 1 GA4GH Tool Execution Challenge",
+       "verified": true
+     }
+   }
+
+Viewing on Dockstore.org
+========================
+
+There are 3 new indicators in Dockstore.org that indicates whether or
+not the tool/workflow is correctly verified.
+
+To see if the verified tool/workflow is correctly displaying to others
+on Dockstore.org, first go to the page of the tool/workflow such as
+https://dockstore.org/containers/quay.io/briandoconnor/dockstore-tool-md5sum:1.0.4?tab=info.
+Since this tool/workflow is correctly verified, the 3 indicators can be
+seen:
+
+.. figure:: /assets/images/docs/verification/tool.png
+   :alt: Tool Page
+
+   Tool Page
+
+1. At the top left, the checkmark indicates that at least one of the
+   tool/workflow’s version has been verified. In general, this
+   tool/workflow is considered verified.
+
+2. At the top right, the recent versions of the tool/workflow are
+   listed. There is a checkmark if a specific version of the
+   tool/workflow is verified.
+
+3. The bottom right shows whether the currently selected/viewed version
+   is verified. The selected version is indicated in the URL as well as
+   the title (e.g. quay.io/briandoconnor/dockstore-tool-md5sum:1.0.4).
+   In this example, the version 1.0.4 is selected/viewed. This bottom
+   right verification box contains more details such as the platform and
+   verifier. This example shows that “Dockstore CLI” is the platform and
+   “Phase 1 GA4GH Tool Execution Challenge” is the verifier.
+
+Addition information for all verified versions can be viewed at a glance
+in the versions tab:
+
+.. figure:: /assets/images/docs/verification/versions-tab.png
+   :alt: Versions Tab
+
+   Versions Tab
+
+Once again, the checkmarks indicate the version is verified. Platforms
+which the version was verified on is displayed to the right of it. In
```

Comment:
are instead of is -- if you remove the which, it's basically Platforms are displayed...